jf_agent/main.py

```diff
@@ -132,6 +132,29 @@ def main():
 
     else:
         jellyfish_endpoint_info = obtain_jellyfish_endpoint_info(config, creds)
+        try:
+            if (
+                jellyfish_endpoint_info.jf_options and
+                'validate_num_repos' in jellyfish_endpoint_info.jf_options.keys() and
+                jellyfish_endpoint_info.jf_options['validate_num_repos']
+            ):
+                validate_num_repos_go_for = True
+            else:
+                validate_num_repos_go_for = False
+        except Exception as e:
+            agent_logging.log_and_print(logger, logging.WARNING, msg=f"Could not get `jf_options` instead got {e}."
+                                                                     f"not critical, continuing.")
+            validate_num_repos_go_for = False
```

Comment:
`jellyfish_endpoint_info` is a dict, right? I think we can get rid of this try catch block if we use the `.get()` call above

Reply:
Oh whoops, disregard my comments. Didn't see it was a `namedtuple`